app/views/docs/realtime.phtml

```diff
@@ -167,12 +194,24 @@ subscription.stream.listen((response) {
     <li>
         <h3>Android</h3>
         <div class="ide margin-bottom" data-lang="android" data-lang-label="Android SDK">
-            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client()
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client(context)
```

Comment:
have we updated everywhere that client requires context in android?

Reply:
Yep 👍